pkg/cmd/scan/output/assets/index.tmpl

```diff
@@ -185,6 +185,11 @@
         <h1 class="congrats">Congrats! Your infrastructure is in sync</h1>
         {{end}}
     </main>
+    <footer>
+        <div>
+            <p>Driftctl scan using provider version {{.ProviderVersion}}</p>
+        </div>
+    </footer>
```

Comment:
I don't think it's the right place for this because if a user has a lot of resources, he'd have to scroll a lot to find that information, isn't it possible to put it in the header or something ?

Reply:
I made something different on the html that he gave me. Will probably push something on top of his PR.